src/core/lwt.mli

```diff
@@ -1916,13 +1911,13 @@ val wakeup_paused : unit -> unit
 
 val paused_count : unit -> int
 (** Returns the number of promises that would be fulfilled by calling
-    {!Lwt.wakeup_paused} right now.
+    [Lwt.wakeup_paused] right now.
```

Comment:
Interesting, the target of the reference as well as the reference itself are removed from the documentation with a stop comment but Odoc still tries to resolve the reference and fails. https://github.com/ocaml/odoc/issues/818